ahf/run/Amiga_profiles.pbs

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#PBS -q condo
+#PBS -N Amiga_h553_spec
+#PBS -l walltime=8:00:00
+#PBS -l nodes=1:ppn=16
+#PBS -o Amiga_out.txt
+#PBS -e Amiga_err.txt
+#PBS -V
+#PBS -M cchoban@ucsd.edu
```

Comment:
You probably don't want to leave your email here. People might submit the job carelessly and spam your email.

Reply:
Good idea